source/change-streams.rst

```diff
@@ -277,33 +306,48 @@ Driver API
 
+    /**
+     * The changeStream will only provides changes that occurred after the
+     * specified cluster timestamp. The default value is a cluster time obtained from
+     * the server before the changestream was created.
+     * @since 4.0
+     */
+    startAtClusterTime: Optional<Timestamp>
+
+    /**
+     * If true, will create a changeStream that operates on the entire cluster.
+     * Is used internally by MongoClient.watch.
+     * @since 4.0
+     */
+    allChangesForCluster: Optional<boolean>
   }
 
 **NOTE:** The set of ``ChangeStreamOptions`` may grow over time.
 
 Helper Method
 -------------
 
-The driver API consists of one helper method located on a driver’s Collection type, as well as a new ``ChangeStream`` type.  The helper MUST return a ``ChangeStream`` instance.  Implementers MUST document that helper method is preferred to running a raw aggregation with a ``$changeStream`` stage, for the purpose of supporting resumability.
+The driver API consists of a ``ChangeStream`` type, as well as three helper methods. All helpers MUST return a ``ChangeStream`` instance. Implementers MUST document that helper methods are preferred to running a raw aggregation with a ``$changeStream`` stage, for the purpose of supporting resumability.
 
-The helper method must construct an aggregation command with a REQUIRED initial ``$changeStream`` stage.  A driver MUST NOT throw a custom exception if multiple ``$changeStream`` stages are present (e.g. if a user also passed ``$changeStream`` in the pipeline supplied to the helper), as the server will return an error.
+The helper methods must construct an aggregation command with a REQUIRED initial ``$changeStream`` stage.  A driver MUST NOT throw a custom exception if multiple ``$changeStream`` stages are present (e.g. if a user also passed ``$changeStream`` in the pipeline supplied to the helper), as the server will return an error.
 
-The helper method MUST determine a read concern for the operation in accordance with the `Read and Write Concern specification <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#via-code>`_.  The initial implementation of change streams on the server requires a “majority” read concern or no read concern.  Drivers MUST document this requirement.  Drivers SHALL NOT throw an exception if any other read concern is specified, but instead should depend on the server to return an error.
+The helper methods MUST determine a read concern for the operation in accordance with the `Read and Write Concern specification <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#via-code>`_.  The initial implementation of change streams on the server requires a “majority” read concern or no read concern.  Drivers MUST document this requirement.  Drivers SHALL NOT throw an exception if any other read concern is specified, but instead should depend on the server to return an error.
```

Comment:
I don't see a difference in this paragraph.  Am I missing it?

Reply:
"The helper method" vs. "The helper methods". Looks like GitHub's diff tool just failed to highlight the change.

Reply:
The change is `s/method/methods/`, since we now have three helpers instead of one helper.